openapi.yml

```diff
@@ -1144,18 +1144,347 @@ paths:
     post:
       summary: Accept block
       description: Accepts a block from the consensus set.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Block"
+      responses:
+        "200":
+          description: Successfully accepted block
+        "500":
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+  /bus/consensus/network:
```

Comment:
we are not quite consistent with newlines between routes. I'd prefer to have one to make sure each route is its own block

Reply:
I was grouping routes. I should've updated the worker routes to be consistent with my rule indeed. I don't mind adding newlines though so whatever you prefer. Updated it.